src/kirby/components/list/list-cell/list-cell.component.ts

```diff
@@ -1,26 +1,103 @@
-import { Component, OnInit, Input, HostBinding } from '@angular/core';
+import {
+  Component,
+  OnInit,
+  Input,
+  HostBinding,
+} from '@angular/core';
 
-type flexAlign = 'flex-start' | 'center' | 'flex-end';
+enum verticalAlignmentEnum {
+  top = 'flex-start',
+  center = 'center',
+  bottom = 'flex-end'
+}
+enum horisontalAlignmentEnum {
+  left = 'flex-start',
+  center = 'center',
+  right = 'flex-end'
+}
+type horisontalAlignment = 'left' | 'center' | 'right';
+type verticalAlignment = 'top' | 'center' | 'bottom';
 
 @Component({
   selector: 'kirby-list-cell',
   templateUrl: './list-cell.component.html',
   styleUrls: ['./list-cell.component.scss']
 })
 export class ListCellComponent implements OnInit {
-  @Input() lineOne: string | number;
-  @Input() lineTwo: string | number;
-  @Input() lineThree: string | number;
-  @Input() alignHorisontal: flexAlign = 'flex-start';
-  @Input() alignVertical: flexAlign = 'flex-start';
-  @Input() flexGrow: string | number;
+  private readonly horisontalAlignmentDefault: horisontalAlignment = 'left';
```

Comment:
If we follow the official angular style guide, place fix the members sequence in [style-05-14](https://angular.io/guide/styleguide#style-05-14)

This also goes for you functions.

Reply:
didn't know that one. It is now fixed :-)